Makefile

```diff
@@ -32,7 +32,8 @@ build:
 	@docker compose --file ${DOCKER_COMPOSE_FILE} build --pull
 
 run:
-	@docker compose --file ${DOCKER_COMPOSE_FILE} up --detach
+	@docker compose --file ${DOCKER_COMPOSE_FILE} up --detach \
+	&& sleep 2 && make unit-reload-config
```

Comment:
For what is that `sleep 2`?

Reply:
It's workaround to reload nginx unit server. Unit configuration base on API, so as you can see in `unit-reload-config` we are using curl.
```shell
unit-reload-config:
	@echo "Reloading Nginx Unit config..." \
	&& docker compose --file ${DOCKER_COMPOSE_FILE} exec ${DOCKER_COMPOSE_APP_CONTAINER} curl -X PUT --data-binary @/application/environment/dev/app/nginx-unit.config.json --unix-socket /var/run/control.unit.sock http://localhost/config
```

Unit needs a while to spin up. So we postpone this for 2 seconds. After compose up, unit is reloading config, so we can be sure that it's working with the current json config.

It's not the best solution but it's simple and it's working.